RAD-Seq/README.md

```diff
@@ -0,0 +1,127 @@
+RAD-Seq data analysis
+======================
+
+RAD-Sequencing is a reduced representation library method used to obtain genomic information on a whole genome without sequencing each nucleotides. Here, we propose some materials to learn how to analyze RAD-Seq data.
+
+# Slides
+
+Several deck of slides are available for this topic:
+
+- [General introduction about RNA seq data analysis](http://bgruening.github.io/training-material/RAD-Seq/slides/)
+- Slide deck related to the tutorials:
+    - [Reference-based RAD-Seq data analysis](http://bgruening.github.io/training-material/RAD-Seq/slides/ref_based.html)
+
+# Tutorials
+
+A tutorial with hands-on is available for this topic:
+
+- [Reference-based RAD-Seq data analysis](tutorials/ref_based.md)
+
+## Input datasets
+
+The input datasets for the tutorials will be soon available on Zenodo.
+
+## Galaxy instance
+
+For these tutorials, you can use the [dedicated Docker image](docker/README.md):
+
+```
+docker run -d -p 8080:80 bgruening/galaxy-RAD-Seq-training
```

Comment:
If you mean the docker related content on the training-material\RAD-Seq\ folder, same answer than for the previous comment, I was planning to add these materials after creating this RAD-seq branch ;) Thaks again again ;)